libafl/src/monitors/tui/mod.rs

```diff
@@ -494,7 +498,7 @@ impl TuiMonitor {
             .get_user_stats("edges")
             .map_or("0%".to_string(), ToString::to_string);
 
-        for client in self.client_stats().iter().skip(2) {
+        for client in self.client_stats().iter().filter(|client| client.enabled) {
```

Comment:
I wonder why this skipped 2 instead of 1? @ToSeven any ideas? Or @andreafioraldi ?

Reply:
Ah think I figured it out